localization.c

```diff
@@ -69,6 +69,22 @@ FindResourceLang(PTSTR resType, PTSTR resId, LANGID langId)
     return FindResource(o.hInstance, resId, resType);
 }
 
+/*
+ * Return value: 0 for LTR, 1 for RTL, 2 or 3 for vertical
+ */
+int
+LangFlowDirection(void)
```

Comment:
Just a nit-pick. It looks like that this function is always used to get the value of `rtl` flag:

    DWORD rtl = (LangFlowDirection() == 1) ? MB_RIGHT|MB_RTLREADING : 0;

Wouldn't it be better to rename it to `GetRTLFlags()` and make it return `MB_RIGHT|MB_RTLREADING` or `0` ?

Reply:
Sure, for use in message boxes that would  be expedient. But we may need flow direction to handle RTL in other Windows like the PLAP task dialog or echo-message. I'll check those and simplify this if there is no other immediate need for a general function like this.